src/tike/view.py

```diff
@@ -80,56 +83,77 @@ def complexHSV_to_RGB( img0 ):
     Returns
     -------
     rgb_img : :py:class:`numpy.array`
-        The (..., 3) shaped array which represents the input complex valued array 
+        The (..., 3) shaped array which represents the input complex valued array
         in a RGB colorspace.
     """
-    
+
     sz = img0.shape
 
-    hsv_img = np.ones( ( *sz, 3 ), 'float32' )
+    hsv_img = np.ones((*sz, 3), 'float32')
+
+    hsv_img[ ..., 0 ] = np.angle( img0 )     
+    hsv_img[ ..., 2 ] = np.abs( img0 )
+
+    #================================
+    # Rescale hue to the range [0, 1]
+
+    eps = np.finfo( np.float32 ).eps
 
-    # Rescale hue and value to the range [0, 1].
-    hsv_img[ ..., 0 ] = np.angle( img0 ) 
     hsv_img[ ..., 0 ] -= np.min( hsv_img[ ..., 0 ] )
-    hsv_img[ ..., 0 ] = hsv_img[ ..., 0 ] / ( np.finfo(np.float32).eps + np.max( hsv_img[ ..., 0 ] ))
+    hsv_img[ ..., 0 ] = hsv_img[ ..., 0 ] / ( eps + np.max( hsv_img[ ..., 0 ] ))
```

Comment:
np.angle always returns values between -PI to PI, so you should use that range instead of min to max.